src/js/utils/permissionUtils.js

```diff
@@ -41,4 +42,13 @@ const canEditRequest = (currentUser, request, location) => {
   return isLocationDestination && isUserRequestor;
 };
 
+export const hasRole = (user, role) => user?.roles?.includes(role);
+
+// To have permissions for product source list actions
+// user has to be at least admin and has product manager permissions
+export const hasPermissionsToProductSourceActions = (user, isAdmin) => {
```

Comment:
I feel like this one can be more generic and reused for other things (or turned into hook)